open-company/open-company-web
# Dropdown menu fixes

Ryan wanted to change the hover state of the dropdown menu.
This change the color to carrot green when a dropdown menu item is hovered together with the icon.

Make sure there is no lag when showing the green icon since it should be preloaded.

Also Stuart wanted the Support link to be in the menu but not in the bottom left corner (below the invite people link).

To test:
- [x] do you not see the Support link in the bottom left corner under invite people? Good
- open dropdown menu
- hover all items
- [x] do you see all the grey and green icons? Good
- [x] does the text turns green when hovering? Good
- click on Support
- [x] did  it open help.carrot.io? Good
- [x] did it open a new window? Good
